docs/pages/examples/playground/footer/index.tsx

```diff
@@ -20,9 +19,7 @@ export const Footer: FC<FooterProps> = memo(({ ...rest }) => {
 
       <IconButton
         variant="ghost"
-        icon={
-          <Icon icon={faClockRotateLeft} fontSize="1.125em" color="muted" />
-        }
+        icon={<History fontSize="2xl" color="muted" />}
```

Comment:
I don't think it's necessary to change the fontSize.

Reply:
@fffk10

That change is intentional. The icon was too small.